src/components/blocks/FormBlock.tsx

```diff
@@ -99,129 +103,129 @@ const FormBlock: React.FC = () => {
         wrapperCol: { offset: 8, span: 16 },
     }
     return (
-        <Form 
+        <Form
             layout='vertical'
             className="form-modal"
+            onFinish={formik.handleSubmit}
         >
             <Form.Item
                 label='Title'
                 name='title'
                 rules={rules.titleRules}
-                initialValue={movie.title}
+                initialValue={formik.values.title}
             >
                 <Input
-                    value={movie.title}
-                    onChange={({ target }) => setMovieByKey({ key: 'title', value: target.value })}
+                    onChange={formik.handleChange}
                     placeholder="example: La La Land"
                 />
             </Form.Item>
             <Form.Item
                 label='Movie average raiting'
                 name='vote_average'
                 rules={rules.vote_average}
-                initialValue={movie?.vote_average}
+                initialValue={formik.values?.vote_average}
```

Comment:
I'm not sure that it will be optional `formik.values.vote_average`

Reply:
removed